ui/raidboss/data/05-shb/trial/diamond_weapon.js

```diff
@@ -0,0 +1,140 @@
+import ZoneId from '../../../../../resources/zone_id';
+import NetRegexes from '../../../../../resources/netregexes';
+import Conditions from '../../../../../resources/conditions';
+import { Responses } from '../../../../../resources/responses';
+
+const sharedOutputStrings = {
+  teleportEast: {
+    en: 'Teleport to east platform',
+  },
+  teleportWest: {
+    en: 'Teleport to west platform',
+  },
+};
+
+export default {
+  zoneId: ZoneId.TheCloudDeck,
+  timelineFile: 'diamond_weapon.txt',
+  triggers: [
+    {
+      id: 'Diamond Diamond Rain',
+      netRegex: NetRegexes.startsUsing({ source: 'The Diamond Weapon', id: '5FA7', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Diamond Claw Swipe East',
+      netRegex: NetRegexes.startsUsing({ source: 'The Diamond Weapon', id: '5F9A', capture: false }),
+      durationSeconds: 10,
+      outputStrings: sharedOutputStrings,
```

Comment:
Sorry for the tiniest of nits, but outputStrings goes last.  I'm honestly surprised the test didn't yell about this ordering.